rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml

```
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+An attacker with Microsoft IIS webserver access via a web shell or alike can decrypt and dump IIS Appool service account
+password using appcmd command. Verify context of this execution such as parent process, file creation, username to
+validate the legitimacy of this action.
```

```suggestion
Identifies the Internet Information Services (IIS) command-line tool, AppCmd, being used to list passwords. An attacker with IIS web server access via a web shell can decrypt and dump the IIS AppPool service account
password using AppCmd. 
```
The below could be moved to `note` - 
> It's recommended to verify the context of this execution, such as parent process name, file creations, and/or username to validate the legitimacy of this action.

---

probably need to lint if you take this recommendation